WpfSendKeys/SendKeysParser.cs

```diff
@@ -2,206 +2,33 @@
 
 namespace System.Windows.Input.Test
```

Comment:
I would really like to change the namespace. Glancing at a class within the `System` namespace makes me assume that this is a well tested and reliable code provided by the framework. It took me a while to realize that it is us who provides the `KeyboardLayout` type.